CO2_Gadget_WIFI.h

```diff
@@ -950,6 +950,9 @@ const char *PARAM_INPUT_1 = "MeasurementInterval";
 const char *PARAM_INPUT_2 = "CalibrateCO2";
 const char *PARAM_INPUT_3 = "SetVRef";
 
+// Predeclaration
+void setDisplayBrightness(uint16_t newBrightness);
```

Comment:
Please, move with all others forward declaration to CO2_Gadget.ino:
https://github.com/melkati/CO2-Gadget/blob/e626693d59fe1f7798dbce906ddfbe1ca08f55ba/CO2_Gadget.ino#L45-L52

Reply:
Of course...
I didn't know you put forwards decalrations there.